test/e2e/framework/.import-restrictions

```diff
@@ -1,12 +1,9 @@
 rules:
   # The core E2E framework is meant to be a normal Kubernetes client,
-  # which means that it shouldn't depend on internal
-  # code. But we are not there yet, so some exceptions
-  # have to be allowed. Over time the list of allowed
-  # packages should get shorter, not longer.
+  # which means that it shouldn't depend on internal code.
   - selectorRegexp: ^k8s[.]io/kubernetes/pkg/
     allowedPrefixes:
-    - k8s.io/kubernetes/pkg/kubelet/apis/
+    # none
```

Comment:
Looks good. I think we have to keep `selectorRegexp: ^k8s[.]io/kubernetes/pkg/`, otherwise using it would be allowed.

Reply:
I take that back. The `selectorRegexp: .` covers `k/k/pkg`.

So this could get removed. Perhaps keep the comment about "is meant to be a normal..." and simply merge it with "The following packages are okay to use"?

Reply:
makes sense to me, thanks. done.